ow_lander/scripts/action_trajectories.py

```diff
@@ -882,6 +882,7 @@ def guarded_move_plan(self, move_arm, robot, moveit_fk, args, server_stop):
         goal_pose.position.z -= direction_z*search_distance
 
         move_arm.set_pose_target(goal_pose)
+        move_arm.set_goal_tolerance(0.05)
```

Comment:
What is the unit for goal tolerance?  If meters, this is about 2 inches.  I have no idea if this is "close enough" according to the sampling location criteria, though it seems reasonable.

Reply:
The terrain varies a lot in the atacama terrain. This was the smallest tolerance that worked for me